packages/dids/src/did.ts

```diff
@@ -94,7 +94,7 @@ export class DidUri {
   params?: Record<string, string>;
 
   /**
-   * Constructs a new `DidUri` instance.
+   * Constructs a new `DidUri` instance from individual components.
```

Comment:
```suggestion
   * Constructs a new `Did` instance from individual components.
```

Reply:
Thanks @LiranCohen.  Went back and found 3 that were missed so pushed up 85ec434